app/webpack/reactjs/components/open_studios/open_studios_info_form.tsx

```diff
@@ -30,6 +32,55 @@ function denullify(participant) {
   return result;
 }
 
+interface SpecialEventScheduleFieldsProps {
+  specialEvent: types.SpecialEventDetails;
+}
+
+interface TimeSlot {
+  start: string;
+  end: string;
+}
+interface TimeSlotCheckBoxProps {
```

Comment:
```suggestion
}

interface TimeSlotCheckBoxProps {
```
You don't have to do this... I'm just a stickler with white-space... 🤦

Reply:
i'm so sloppy with vertical white space.  but happy to add it in.